streaming/stream_session.cc

```diff
@@ -200,6 +200,9 @@ void stream_manager::init_messaging_service_handler(abort_source& as) {
                     auto distribute = [this, cf_id, s] (flat_mutation_reader_v2 producer,
                             std::function<future<> (flat_mutation_reader_v2)> consumer) -> future<received_partitions_t> {
                         try {
+                            utils::get_local_injector().inject("stream_mutation_fragments_table_dropped", [this] () {
```

Comment:
Since we are at it, could you add such injection in the sender side as well?

Reply:
Turned out to fail, I would look at it as a follow up